.github/workflows/scripts/generate-playground-blueprint.js

```diff
@@ -70,9 +70,7 @@ async function run({ github, context }) {
 ## Test using WordPress Playground
 The changes in this pull request can be previewed and tested using a [Wordpress Playground](https://playground.wordpress.net/playground/) instance.
 
-[Test this pull request with Wordpress Playground](${url}).
-
-Note that this URL is valid for 30 days from when this comment was last updated. You can update it by closing/reopening the PR or pushing a new commit.`;
```

Comment:
I decided to remove this for now as I don't fully understand why it would expire in 30 days. @samueljseay could you elaborate on why you added this warning? I can't see why/how a Playground link would expire and how retriggering the flow would reactivate it 🤔

Reply:
OK, I assume it has to do with some peculiarities of how Woo builds are built, as the blueprint for Gutenberg is a bit simpler. Given we don't need to expire here (from what I see), I think I can optimize the script by just skipping the creation of a comment if it has already been found instead of updating it. I'll do that tomorrow.